### PR TITLE
Updated postgres doc and fixed other minor nits

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Updated the Postgres doc, the connector catalog page, and the Glassfrog doc

### CODE REVIEW

File: docs/integrations/sources/postgres.md [hunk 1]
```diff
@@ -238,7 +238,22 @@ Also, the publication should include all the tables and only the tables that nee
 The Airbyte UI currently allows selecting any tables for CDC. If a table is selected that is not part of the publication, it will not be replicated even though it is selected. If a table is part of the publication but does not have a replication identity, that replication identity will be created automatically on the first run if the Airbyte user has the necessary permissions.
 :::
 
-#### Step 5: Set up the Postgres source connector
+#### Step 5: [Optional] Set up initial waiting time
```
Comment (Amruta-Ranade) [inline docs/integrations/sources/postgres.md:241]: @tuliren Let me know if this looks right!

File: docs/integrations/sources/postgres.md [hunk 2]
```diff
@@ -238,7 +238,22 @@ Also, the publication should include all the tables and only the tables that nee
...
+:::warning
+This is an advanced feature. Use it if absolutely necessary.
+:::
+
+The Postgres connector may need some time to start processing the data in the CDC mode in the following scenarios:
+
+- When the connection is set up for the first time and a snapshot is needed
+- When the connector has a lot of change logs to process
+
+The connector waits for the default initial wait time of 5 minutes (300 seconds). Setting the parameter to a longer duration will result in slower syncs, while setting it to a shorter duration may cause the connector to not have enough time to create the initial snapshot or read through the change logs.
```
Comment (tuliren) [inline docs/integrations/sources/postgres.md:252]: We added a limit on the min and max values for this field for safety as follows:

```suggestion
The connector waits for the default initial wait time of 5 minutes (300 seconds). Setting the parameter to a longer duration will result in slower syncs, while setting it to a shorter duration may cause the connector to not have enough time to create the initial snapshot or read through the change logs. The valid range is 120 seconds to 1200 seconds.
```

**Review by tuliren — APPROVED** [review]
Looks good except for one comment!

File: docs/integrations/sources/postgres.md (hunk 2 above)
Comment (Amruta-Ranade) [reply docs/integrations/sources/postgres.md]: Done!

Comment (tuliren) [reply docs/integrations/sources/postgres.md]: Thank you.

**Pull request merged**